client/exporters.go

```diff
@@ -1,9 +1,10 @@
 package client
 
 const (
-	ExporterImage  = "image"
-	ExporterLocal  = "local"
-	ExporterTar    = "tar"
-	ExporterOCI    = "oci"
-	ExporterDocker = "docker"
+	ExporterImage      = "image"
+	ExporterLocal      = "local"
+	ExporterTar        = "tar"
+	ExporterOCI        = "oci"
+	ExporterDocker     = "docker"
+	ExporterNydusImage = "nydus"
```

Comment:
Do we really need a new exporter for nydus? Can we add "nydus" as a "compression format" for existing exporters? Please see https://github.com/moby/buildkit/pull/2057.

Reply:
Hi, here is an [example](https://github.com/dragonflyoss/image-service/blob/master/docs/nydus-design.md#2-image-manifest) of Nydus image. Nydus image is not currently a standard OCI image format, it is composed of a bootstrap layer and some blob layers, so we can't just treat the format of these layers as another compression type.

Reply:
We can rename "compression type" to another word as you like, e.g.,  "blob type" , but having another exporter is fine to me as well

@ktock WDYT?

Reply:
Based on #2057, the manifest index converter [is configurable](https://github.com/containerd/containerd/blob/f201b78b9065f167071bf472a9708b3921c3d5d1/images/converter/converter.go#L64-L71) so nydus image should be able to generated without introducing a new exporter.

Reply:
Thanks, that's a good point if using a configurable converter, we will take a look.